docs/v1/P4Runtime-Spec.mdk

```diff
@@ -6071,6 +6072,11 @@ man-in-the-middle attacks between the server and client.
 
 ## Revision History
 
+### Changes in v1.3.0
+
+* Deprecate `watch` field (uint32) in favor of `watch_port` (bytes). This allows
```

Comment:
nit: the previous type was int32, not uint32

Reply:
Fixed.